test/integration/config.ts

```diff
@@ -0,0 +1,92 @@
+import { Wallet } from 'ethers'
```

Comment:
Do the benchmark tests and other test written in JS work? 

At least `npm run benchmarks` command fails as `publish.js` JS-file imports to a `config.ts` TypeScript file. Maybe we need to convert all tests to TypeScript, or import the built JS file from `dist` folder.

Reply:
I'm unable to run any tests because of babel errors

Reply:
but that's a good point. I'll keep the old js file and just make the ts file under different name. Good catch.

Reply:
seems IDEA refactors don't spot that kind of problems  :x  worrying.